hugo/content/documentation/administration/acl/index.md

```diff
@@ -0,0 +1,250 @@
+
+Groups and permissions are configured on channels. Channels inherit groups and rules by default. Groups can be used for grouping users into user roles. ACL rules are used to give or deny users or groups of users permissions on a channel.
+
+### User Groups
+
+In the *Groups* tab, you can see and edit the user groups of this channel.
+
+![Screenshot of channel groups dialog](screenshot-channel-groups.jpg)
+
+For a simple configuration, you will edit user groups on your servers root channel (the top-level, outer-most channel) and inherit these groups to all channels.
+
+![Screenshot of the root channel context menu with edit action](edit-root-channel.jpg)
+
+To describe the default permission setup we refer to the following example channel tree:
+
+```text
+Root
+  ChannelA1
+    ChannelA11
+    ChannelA12
+    ChannelA13
+  ChannelB
+```
+
+By default
+
+* the `Root` channel defines an `admin` group
+  * the `admin` group is inheritable
+* the other channels inherit the `admin` group
+
+#### Advanced User Groups
+
+Because user groups can be defined in channels you can define separate user groups for different channel sub trees, individual channels, or influence who is a member of a group in the context of different channels.
+
+For example, to have different moderators users in separate channel trees, given the following channel setup
+
+```text
+Root
+  ChannelA1
+    ChannelA11
+    ChannelA12
+    ChannelA13
+  ChannelB
+```
+
+we can define a group `moderatorA` in `ChannelA1` and a group `moderatorB` in `ChannelB`.
+
+We may also define a group `userA` in `ChannelA1` and a group `userB` in `ChannelB` intending to only allow `userA` members to enter `ChannelA1` and its children, while only allowing `userB` members to enter the `ChannelB` channel.
+
+## ACL rules
+
+ACL rules are configured in the ACL tab of the channel edit dialog.
+
+![Screenshot of ACL dialog](screenshot-channel-acl.jpg)
+
+The Access Control *List* can have multiple rules. Rules may be inherited from parent channels - displayed in italics - or defined on this channel.
+
+Permission are evaluated from top to bottom. (*Allow* and *deny* may be overwritten further down.)
+
+### Selector
+
+A rule consists of a selector and permission actions.
+
+The **selector defines who the rule applies to** - a user, user group, or password group. It may use a prefix to indicate its type or modify its select-behavior.
+
+| Prefix | Prefix-Type   | Description                                      | Example                        |
+| ------ | ------------- | ------------------------------------------------ | ------------------------------ |
+| *none* | Id-Type       | User                                             | `User1`, `John Doe`            |
+| `@`    | Id-Type       | Group                                            | `@admin`, `@all`, `@moderator` |
+| `@#`   | Id-Type       | Channel password / Access Token                  | `@#secret`, `@#u8DhwG2`        |

```

Comment:
The rule is displayed with `@#token` in the ACL rule list.

When you select it, the group field shows value `#token`.

This is the same behavior as `@all` in the list being shown as `all` in the group field.

Describing it as `@#token` here is consistent to describing `@all` rather than `all`.

I will resolve this as correct.